magnet/Source/CommandHandler.cpp

```diff
@@ -613,7 +613,7 @@ namespace MG
 		auto ifTrue = [&]()
 		{
 			emitter.Add_Indentation();
-			emitter.Add_SetTargetProperties(projectName, "LINK_FLAGS", "-Wl, -rpath, ./");
+			emitter.Add_SetTargetProperties(projectName, "\"LINK_FLAGS", "-Wl,-rpath,./\"");
```

Comment:
Shouldn't it be `\"-Wl,-rpath,./\"` , since we only need to provide quotations around the flags?

Reply:
Whoops, I had to do some reverting due to the formatter changing the whole file and on the second time around I made a nasty mistake, I'll correct it right away

Reply:
there, it should be fixed now